database_updates.php

```diff
@@ -1636,11 +1638,55 @@
+            `inventory_id` int(11) NOT NULL AUTO_INCREMENT,
+            `inventory_product_id` int(11) NOT NULL,
+            `inventory_location_id` int(11) NOT NULL,
+            `inventory_client_id` int(11) DEFAULT NULL,
+            `inventory_ticket_id` int(11) DEFAULT NULL,
+            `inventory_notes` text DEFAULT NULL,
+            `inventory_vendor_id` int(11) DEFAULT NULL,
+            `inventory_quantity` int(11) NOT NULL DEFAULT 0,
+            `inventory_cost` decimal(15,2) NOT NULL DEFAULT 0.00,
+            `inventory_created_at` datetime NOT NULL DEFAULT current_timestamp(),
+            `inventory_updated_at` datetime DEFAULT NULL ON UPDATE current_timestamp(),
+            `inventory_archived_at` datetime DEFAULT NULL,
+            PRIMARY KEY (`inventory_id`)
+          ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;");
+
+        mysqli_query($mysqli, "CREATE TABLE `inventory_locations` (
+            `inventory_location_id` int(11) NOT NULL AUTO_INCREMENT,
+            `inventory_location_name` varchar(200) NOT NULL,
+            `inventory_location_description` text DEFAULT NULL,
+            `inventory_location_address` varchar(200) DEFAULT NULL,
+            `inventory_location_city` varchar(200) DEFAULT NULL,
+            `inventory_location_state` varchar(200) DEFAULT NULL,
+            `inventory_location_zip` varchar(200) DEFAULT NULL,
+            `inventory_location_country` varchar(200) DEFAULT NULL,
+            `inventory_location_created_at` datetime NOT NULL DEFAULT current_timestamp(),
+            `inventory_location_updated_at` datetime DEFAULT NULL ON UPDATE current_timestamp(),
+            `inventory_location_archived_at` datetime DEFAULT NULL,
+            `inventory_location_user_id` int(11) NOT NULL DEFAULT 0,
+            PRIMARY KEY (`inventory_location_id`)
+          ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci;
+          ");
+
+        mysqli_query($mysqli, "CREATE TABLE `ticket_products` (
+            `ticket_product_id` int(11) NOT NULL AUTO_INCREMENT,
+            `ticket_product_product_id` int(11) NOT NULL,
+            `ticket_product_ticket_id` int(11) NOT NULL,
+            `ticket_product_quantity` int(11) NOT NULL,
+            PRIMARY KEY (`ticket_product_id`)
+          ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb3 COLLATE=utf8mb3_general_ci");
+
+        mysqli_query($mysqli, "CREATE TABLE `credits ` (`credit_id` int(11) NOT NULL AUTO_INCREMENT,`credit_amount` decimal(15,2) NOT NULL,`credit_currency_code` varchar(200) NOT NULL,`credit_date` date NOT NULL,`credit_reference` text DEFAULT NULL,`credit_created_at` datetime NOT NULL DEFAULT current_timestamp(),`credit_updated_at` datetime DEFAULT NULL ON UPDATE current_timestamp(),`credit_archived_at` datetime DEFAULT NULL, `credit_client_id` int(11) NOT NULL,`credit_payment_id` int(11) NOT NULL,`credit_account_id` int(11) NOT NULL, PRIMARY KEY (`credit_id`))");  
```

Comment:
There is an extra space after the word credits preventing the creation of the table during the database update.